src/sonic-config-engine/minigraph.py

```diff
@@ -157,10 +190,19 @@ def parse_png(png, hname):
                             elif node.tag == str(QName(ns, "EndDevice")):
                                 mgmt_dev = node.text
 
-    return (neighbors, devices, console_dev, console_port, mgmt_dev, mgmt_port, port_speeds, console_ports)
+    return (neighbors, devices, console_dev, console_port, mgmt_dev, mgmt_port, port_speeds, console_ports, device_neighbors)
+
 
+def parse_dpg(dpg, hname, device_hostname):
 
-def parse_dpg(dpg, hname):
+    #In Multi-NPU platforms the acl intfs are defined for only for the host.
+    # to get the aclintfs node first.
```

Comment:
can you standard docstring to add comments?

https://www.python.org/dev/peps/pep-0257/

Reply:
Updated as per comment.